_posts/2022-03-02-getting-started-project-flotta.markdown

```diff
@@ -30,7 +33,27 @@ $ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/downloa
 $ kubectl wait --for=condition=Ready pods --all -n cert-manager --timeout=60s
 ```
 
-To deploy the latest version of the `flotta` operator execute following command:
+In addition, OpenShift router is needed for passing-through incoming TLS traffic into Flotta Edge API.
+For non-OCP clusters types(k8s, kind, minikube), use the following command to install it:
+```bash
+$ kubectl apply -f https://raw.githubusercontent.com/openshift/router/master/deploy/router_rbac.yaml
+$ kubectl apply -f https://raw.githubusercontent.com/openshift/router/master/deploy/route_crd.yaml
+$ kubectl apply -f https://raw.githubusercontent.com/openshift/router/master/deploy/router.yaml
+$ kubectl wait --for=condition=Ready pods --all -n openshift-ingress --timeout=60s
+```
+
+[Noobaa](https://noobaa.github.io/) is data service that provides S3 object-store interface. Flotta's Edge API enables
+edge devices to send application data to S3 endpoint, into a bucket claim created automatically for each device at
+registration time.
+
+For Installing [Noobaa](https://noobaa.github.io/) on the cluster, follow [Noobaa installation guide]([Noobaa](https://github.com/noobaa/noobaa-operator)).
+Note that noobaa requires additional resources, therefore additional memory and CPU are needed(+2 cpu, +5Gi memory).
+Noobba also requires a local storage class and a default storage class to be present on the cluster.
+
+In order to skip the _Noobaa_ installation, follow the steps below to disable Object Bucket claim auto-creation.
```

Comment:
I like the use of italic for _Noobaa_ but I think we should use it for other "Noobaa" occurrences as well